ci/docker/docker-in-docker-image/publish-docker-image/publish_docker_image/emulator.py

```diff
@@ -23,7 +23,10 @@ def publish_emulator(directory, registry, image, versions):
         not_prepared_image = build_image(
             image=full_image,
             directory=directory,
-            build_args={'SDK_VERSION': version}
+            build_args={
+                'SDK_VERSION': version,
+                'EMULATOR_ARCH': "x86" if int(version) < 28 else "x86_64"
```

Comment:
Can we use x64 images for pre 28 API also?

Reply:
I don't understand what profit will we get?

Reply:
The main hypothesis is performance: https://stackoverflow.com/a/47451413/1282707 
But we need benchmarks for emulators.